src/commands/env/create/compute.ts

```diff
@@ -76,36 +76,23 @@ export default class EnvCreateCompute extends Command {
       // This query allows us to verify that the org connection has actually been created before
       // attempting to create a compute environment. If we don't wait for this to happen, environment
       // creation will fail since Heroku doesn't yet know about the org
-      let response: QueryResult<FunctionConnectionRecord>;
-
-      try {
-        response = await connection.query<FunctionConnectionRecord>(`SELECT
+      const start = new Date().getTime();
+      const response: QueryResult<FunctionConnectionRecord> = await connection.query<FunctionConnectionRecord>(`SELECT
           Id,
           Status,
           Error
-          FROM FunctionsConnection`);
-      } catch (err) {
-        const error = err as Error;
-        // This is obviously heinous, but should only exist short-term until the move from `FunctionsConnection`
```

Comment:
Thanks for cleaning that up!

Reply:
yah - obsolete code that didn't need to be there after early GA release.